modules/upi-provisioning-new-hosts-in-a-upi-cluster.adoc

```diff
@@ -68,26 +179,30 @@ $ oc create -f bmh.yaml
 .Example output
 [source,terminal]
 ----
-secret/worker1-bmc created
-baremetalhost.metal3.io/worker1 created
+secret/openshift-worker-<num>-network-config-secret created
```

Comment:
Add a callout to describe <num>?

Reply:
It's example output, not something the end user needs to input via yaml.